components/help-and-support/BrowseTopicSection.js

```diff
@@ -111,8 +111,8 @@ const BrowseTopics = () => {
         <Grid
           justifyContent="center"
           alignItems="center"
-          width={['100%', '568px', '864px', null, null, '1020px']}
-          gridTemplateColumns={[null, 'repeat(2, 288px)', 'repeat(3, 272px)', null, `repeat(3, 320px)`]}
+          width={['100%', '568px','750px', '864px', null, null, '1020px']}
+          gridTemplateColumns={[null, 'repeat(2, 288px)', 'repeat(2, 300px)','repeat(3, 272px)', null, `repeat(3, 320px)`]}
```

Comment:
We only support [4 breakpoints](https://github.com/opencollective/opencollective-frontend/blob/0868babd6bbd5a667bc11dc75340a77c316804d1/lib/theme/breakpoints.ts). So only the 4 first values will be considered. The `1020px` in there was already unnecessary, but with the addition of `750px` we're not considering `null` anymore, which disabled the fixed width and is likely the root cause for the missing gap reported above.